Paws4Me/Paws4Me/Controllers/Logic/ViewController.swift

```diff
@@ -8,109 +8,227 @@
 import UIKit
 
 class ViewController: UIViewController, UISearchBarDelegate {
+    // MARK: - IBOulets
     @IBOutlet weak private var petTable: UITableView!
     @IBOutlet weak private var searchBar: UISearchBar!
-    private var filteredPetObject: AdoptPet?
-    private var adoptPetObject: AdoptPet?
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        getAdoption()
-        petTable.delegate = self
-        petTable.dataSource = self
-        self.title = "Adoptable Animals"
-        setUpSearchbar()
-    }
+    @IBAction func btnCatTapped(_ sender: UIButton) {
```

Comment:
Please also adjust this structure based on previous comment 😄

Reply:
Noted. Thank you for the advice👍🏻